pypdf/_writer.py

```diff
@@ -1242,14 +1253,13 @@ def write_stream(self, stream: StreamType) -> None:
                 "It may not be written to correctly.",
                 __name__,
             )
+        # no more used :
+        # if not self._root:
+        #   self._root = self._add_object(self._root_object)
+        # self._sweep_indirect_references(self._root)
```

Comment:
Commented code does not make sense with Git.

Reply:
I left it for understanding where _sweep_indirect_references was used